tests/error001.stderr

```diff
@@ -1,5 +0,0 @@
-error001.y: Multiple rules for 'foo'
```

Comment:
Oh sorry, I forgot to ask: shouldn't this go somewhere?

Reply:
Yes, but I hesitated to reproduce the logic for running error-tests (i.e. test that are expected to fail) and comparing their output with some previously defined output, given that there is only a single one of these error-tests which is additionally very trivial.

I believe it it okay to drop `error001.y`.

Should more of these error-tests be written, `happy-test` can be extended accordingly.